packages/block-editor/src/components/link-control/link-preview.js

```diff
@@ -28,9 +28,9 @@ export default function LinkPreview( {
 	hasRichPreviews = false,
 } ) {
 	// Avoid fetching if rich previews are not desired.
-	const maybeRemoteURL = hasRichPreviews ? value?.url : null;
+	const useRichPreviews = hasRichPreviews ? value?.url : null;
```

Comment:
Extremely minor, but I think it'd be good to reserve the `use` prefix for react hooks as much as possible.

Reply:
Not minor at all. Needs fixing. Will do that now.